data/data-pipeline/data_pipeline/config.py

```diff
@@ -12,7 +12,7 @@
 
 # set root dir
 settings.APP_ROOT = pathlib.Path(data_pipeline.__file__).resolve().parent
-
+settings.REQUESTS_DEFAULT_TIMOUT = 3600
```

Comment:
Since before we had no timeouts, which means wait forever, I picked a very generous hour. This was arbitrary tho --- of course open to changing it.